.github/workflows/test.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       matrix:
         os:
           - ubuntu-x64-small
-        node: [20.x, 22.x, 24.x]
+        node: [20.x, 22.x, 24.x, 25.x]
```

Comment:
Please add `25.x` to this line too:

https://github.com/grafana/pyroscope-nodejs/blob/6c9c34f00e8b1846d251b4d320d8db0de13fd3f4/.github/workflows/test.yml#L37